src/main/java/me/jellysquid/mods/sodium/client/gui/prompt/ScreenPrompt.java

```diff
@@ -74,7 +74,7 @@ public void render(DrawContext drawContext, int mouseX, int mouseY, float delta)
             textY += 8;
         }
 
-        this.closeButton = new FlatButtonWidget(new Dim2i((boxX + width) - 84, (boxY + height) - 24, 80, 20), Text.literal("Close"), this::close);
+        this.closeButton = new FlatButtonWidget(new Dim2i((boxX + width) - 84, (boxY + height) - 24, 80, 20), Text.translatable("sodium.options.buttons.close"), this::close);
```

Comment:
I could've gone with `gui.done` but it didn't seem appropriate in some error or warning popups